src/WinUI/ArcGIS.WinUI.Viewer/Samples/Layers/AddSceneLayerWithElevation/readme.metadata.json

```diff
@@ -17,16 +17,18 @@
     ],
     "offline_data": [],
     "redirect_from": [
-        "/net/latest/wpf/sample-code/scene-layer-url.htm"
-    ],
+        "/net/latest/winui/sample-code/add-scene-layer-with-elevation.htm"
```

Comment:
I don't think we need the "latest" string in redirect link?

Reply:
@williambohrmann3 Fixed, should we still wait on Hamish's input on the paths?

Reply:
@williambohrmann3 Interesting OK. Well I got rid of the `latest`, now I'll just add in the old sample redirect path as well. 

This worries me a bit because I've had quite a few sample re-names/consolidations that have been merged recently and the previous redirect path was NOT included. I should open an issue to go back and fix that if this is the case.